container_images/cli-tools-module-tests/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Copyright 2021 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FROM gcr.io/gcp-guest/gotest
+
+COPY gcsfuse-apt-key.gpg .
```

Comment:
i dont think we should be storing binary artifacts in our repos, esp if we dont produce them. is there a canonical source this can be downloaded from, instead?

Reply:
There is a canonical source. 

1. https://packages.cloud.google.com/apt/doc/apt-key.gpg
2. https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/docs/installing.md

There are two problems with downloading it during build: it may not be available, and it may not be the artifact we're expecting. By storing it here we avoid both. 

People who download during build hardcode a checksum and compare it. I'm okay with that, but given how small the key is I'm not sure of the benefit.

Reply:
this is only going to happen during build, it cant affect a production system. so i think we should use the canonical source, with or without checksum

Reply:
done!